app/models/concerns/has_participations_to_rdvs.rb

```diff
@@ -41,9 +41,25 @@ def rdvs?
     !rdvs.empty?
   end
 
-  def rdv_seen_delay_in_days
+  def days_between_follow_up_creation_and_first_seen_rdv
     return if first_seen_rdv_starts_at.blank?
 
     first_seen_rdv_starts_at.to_datetime.mjd - created_at.to_datetime.mjd
   end
+
+  def days_between_first_orientation_seen_rdv_and_first_seen_rdv
+    return unless first_orientation_seen_rdv_date && first_seen_rdv_starts_at
```

Comment:
On pourrait d'abord checker s'il y a des rdvs seen avant de récupérer la date du rdv d'orientation. Et pour plus de lisibilité: 
```suggestion
    return unless seen_rdvs? 
    return unless first_orientation_seen_rdv_date
```

Reply:
C'est plus facile à lire oui !